src/config/maps/_config-core-maps.scss

```diff
@@ -7,5 +7,6 @@
 @import "core/core-radius-map";
 @import "core/core-shadows-map";
 @import "core/core-common-map";
-@import "core/core-fonts-map";
+@import "core/core-fonts-map-deprecated";
```

Comment:
maybe worth keeping it `core-fonts-map` and to create the new one as `core-fonts-map-new`.
just because in some places you have `-new` (e.g. `$core-typography-fonts-new-map`) and in some you have deprecated

Reply:
changing everything to deprecated then :)